### PR TITLE
pyside 6.2.2

### PR DESCRIPTION
**Pull request by paperchalice**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/HEAD/CONTRIBUTING.md)?
- [x] Have you ensured that your commits follow the [commit style guide](https://docs.brew.sh/Formula-Cookbook#commit)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [ ] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [ ] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [ ] Does your build pass `brew audit --strict <formula>` (after doing `brew install --build-from-source <formula>`)? If this is a new formula, does it pass `brew audit --new <formula>`?

-----
Add `packaging` during installation.

### CODE REVIEW

File: Formula/pyside.rb [hunk 1]
```diff
@@ -28,14 +30,23 @@ class Pyside < Formula
...
+  end
+
   def install
     # upstream issue: https://bugreports.qt.io/browse/PYSIDE-1684
     inreplace "sources/pyside6/cmake/Macros/PySideModules.cmake",
               "${shiboken_include_dirs}",
               "${shiboken_include_dirs}:#{Formula["qt"].opt_include}"
 
+    python = Formula["python@3.9"]
+    venv = virtualenv_create(buildpath/"venv", python.opt_bin/"python3")
```
Comment (carlocab) [inline Formula/pyside.rb:45]:
```suggestion
    venv = virtualenv_create(buildpath/"venv", "python3")
```

File: Formula/pyside.rb [hunk 2]
```diff
@@ -28,14 +30,23 @@ class Pyside < Formula
...
+    venv.pip_install resources
+
     qt = Formula["qt"]
-    site_packages = prefix/Language::Python.site_packages("python3")
+    site_packages = prefix/Language::Python.site_packages(python.opt_bin/"python3")
```
Comment (carlocab) [inline Formula/pyside.rb:49]:
```suggestion
    site_packages = prefix/Language::Python.site_packages("python3")
```

File: Formula/pyside.rb [hunk 3]
```diff
@@ -28,14 +30,23 @@ class Pyside < Formula
   uses_from_macos "libxml2"
   uses_from_macos "libxslt"
 
+  resource "packaging" do
```
Comment (carlocab) [inline Formula/pyside.rb:33]:
```suggestion
  uses_from_macos "libxslt"

  fails_with gcc: "5"

  resource "packaging" do
```

**Review by carlocab — DISMISSED** [review]
Thanks!

File: Formula/pyside.rb (hunk 1 above)
Comment (carlocab) [reply Formula/pyside.rb]: Looks like this was needed. That's unexpected. Apologies for the bad suggestion!

Comment (paperchalice) [reply Formula/pyside.rb:47]: Never mind.😁

**Review by carlocab — APPROVED** [review]
Looks like the Linux build requires a bit more work, but that can be fixed separately.

**Pull request closed without merging**